test/tests/outputs/postprocessor/output_pps_hidden_shown_check.i

```diff
@@ -0,0 +1,90 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 10
+  ny = 10
+  elem_type = QUAD4
+  # This test uses ElementalVariableValue postprocessors on specific
+  # elements, if you use DistributedMesh the elements get renumbered.
+  parallel_type = replicated
```

Comment:
I do not know. I admit this is a copy and paste. But there are comments in the code that if we use `ElementalVariableValue` we need to run with `parallel_type = replicated`. I am guessing this is needed for the parallel target, i.e. this test will not be run...